mitiq/mitiq_qiskit/tests/test_qiskit_utils.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (C) 2020 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Unit tests for qiskit executors (qiskit_utils.py)."""
+import pytest
+import numpy as np
+from qiskit import QuantumCircuit
+
+from mitiq.mitiq_qiskit.qiskit_utils import (
+    execute,
+    execute_with_shots,
+    execute_with_depolarizing_noise,
+    execute_with_shots_and_depolarizing_noise,
+)
+
+OBSERVABLE = np.array([[1, 0], [0, 0]])
```

Comment:
```suggestion
ONE_QUBIT_OBSERVABLE = np.array([[1, 0], [0, 0]])
```

Reply:
Perhaps even better to call these `ONE_QUBIT_GS_PROJECTOR` and `TWO_QUBIT_GS_PROJECTOR` to make the tests more readable. Ignore if they're too long.